packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -123,7 +164,7 @@ export class MonacoEditorProvider {
         }
     }
 
-    protected installQuickOpenService(editor: MonacoEditor): void {
+    protected installQuickOpenService(editor: MonacoEditor | MonacoDiffEditor): void {
```

Comment:
Why is that? As I see it is a subclass of the `MonacoEditor`. Is it just some leftover code?